pennylane/configuration.py

```diff
@@ -90,7 +91,7 @@
 ``[plugin.global]``.
```

Comment:
Just leaving this as a note here, but should we update projectQ plugin so `num_runs` is replaced by `shots`?